libcudacxx/include/cuda/std/__cccl/builtin.h

```diff
@@ -508,4 +508,22 @@
 #  define _CCCL_BUILTIN_UNDERLYING_TYPE(...) __underlying_type(__VA_ARGS__)
 #endif // _CCCL_CHECK_BUILTIN(underlying_type) && gcc >= 4.7
 
+#if defined(_CCCL_COMPILER_MSVC)
+#  if _CCCL_MSVC_VERSION >= 1935
```

Comment:
This is available sind 19.27
```suggestion
#  if _CCCL_MSVC_VERSION >= 1927
```

Reply:
in my testing, it is available starting 19.35. https://godbolt.org/z/Pnso6o89G

Reply:
Oh yeah I was thinking about `__builtin_FUNCTION`